.gitignore

```diff
@@ -17,3 +17,5 @@ tests/freetype/objs/*.lo
 third_party/lzma.js/lzip/*.o
 third_party/lzma.js/lzma-native
 third_party/lzma.js/lzma-native.exe
+
```

Comment:
Yes, you are right! I added this line into .gitignore in my local branch because it bothers me to see git reports it as 'Untracked files' each time I issue ```git status``` command. I then accidentally include it in this commit when I did ```git ci -a```. But then I think there is no harm to have this change as well. But you are right, it is not related. If it bothers you then I could rebase the commit history in this PR to exclude it.